### DISCUSSION TITLE
Volume knob is too sensitive

### DISCUSSION THREAD

**Discussion by cYCL157** [rern/rAudio#1715] [Bug Reports]
I have been using rAudio/RuneAudio for years and have loved it!  I have an iPhone and have saved the web view to my home screen and use it like an "app".  That has worked well for me with an exception that seemed to start with the 20240707 release where the volume knob has become "too sensitive" -- meaning that I often turn up the volume accidentally when I tried to switch apps by using the swipe up from the bottom of the screen gesture.  I just had it happen as I was holding my phone and setting it down on the my charger as I switch my hand position on the phone touching the bottom right corner.  It seems that whatever I am accidentally doing is trying to set the volume to 100.  This often happens after i have swiped up and gone to home screen or other app and rAudio "app" is in background and i have to quickly switch back to turn down the volume.

I don't think my handling and use of the UI has changed, but i am much more careful to try to not swipe on the volume knob but i often still catch it.

Attached is a screenshot of how i have the app configured with the view i get.  I have an iPhone 15 Pro, if that matters.

Love rAudio!

Cort

![2024-07-19 21 19 39](https://github.com/user-attachments/assets/8317a9e9-3521-4dac-a63d-c97c3ccdeed4)

**Comment by ceejayemm** [comment]
@cYCL157 

I agree, I currently have 20240719 installed on and RPi3B+ and access it via a web browser or the RA app on an Android phone.  Right now the volume is show as 0 (zero) but the system is playing at normal audio levels.  I can adjust the volume level, in the app or on the browser, but initially I either get zero or max volume and then have to be extremely accurate to get back to the level I want.  The next time I look the displayed volume level will be down as zero again with the system actually playing as normal.  The volume control seems overly sensitive in its current incarnation.

Chris
